### PR TITLE
Remove SQLite warnings in integration tests

### PR DESCRIPTION
**Pull request by IgorChernyshov**
SQLITE_FILEPROTECTION warnings were spammed be URLSession default configuration which tries to write cookies and other things into cache. I believe tests should not relate on saved cache data, so I used URLSession with ephemeral configuration that keeps everything in operating memory and doesn't try to store things on disk.

### CODE REVIEW

**Comment by bidoubiwa** [comment]
While the tests pass locally they seem to break on the CI due to a timeout after 10. Any idea why with your addition tests are so slow to start?

**Comment by IgorChernyshov** [comment]
@bidoubiwa I'm sorry, I can't reproduce the issue.

**Comment by curquiza** [comment]
bors try

**Comment by curquiza** [comment]
Seems like you can merge it @bidoubiwa

File: Tests/MeiliSearchIntegrationTests/UpdatesTests.swift [hunk 1]
```diff
@@ -32,14 +32,15 @@ class UpdatesTests: XCTestCase {
...
   private let uid: String = "books_test"
 
   // MARK: Setup
 
   override func setUp() {
     super.setUp()
-
-    client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey")
+	session = URLSession(configuration: .ephemeral)
+	client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey", session: session)
```
[the remark this answers is not in the document]
Comment (curquiza) [reply Tests/MeiliSearchIntegrationTests/UpdatesTests.swift:43]:
```suggestion
	  session = URLSession(configuration: .ephemeral)
	  client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey", session: session)
```

File: Tests/MeiliSearchIntegrationTests/UpdatesTests.swift [hunk 2]
```diff
@@ -32,14 +32,15 @@ class UpdatesTests: XCTestCase {
...
   private let uid: String = "books_test"
 
   // MARK: Setup
 
   override func setUp() {
     super.setUp()
-
-    client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey")
+	  session = URLSession(configuration: .ephemeral)
+	  client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey", session: session)
```
Comment (curquiza) [inline Tests/MeiliSearchIntegrationTests/UpdatesTests.swift:43]:
```suggestion
	   session = URLSession(configuration: .ephemeral)
	   client = try! MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey", session: session)
```

**Review by curquiza — APPROVED** [review]

**Comment by curquiza** [comment]
bors try

**Pull request merged**